### ISSUE TITLE
SEO image upload failed. Use Feed Me to import entries data.

### ISSUE THREAD

**Issue by Owen-Guo** [nystudio107/craft-seomatic#1554]
When using Feed Me to import Entry data, select custom image as Image Source as shown in the screenshot and enter the resource link in custom image. The entry is created successfully, but the SEO image does not successfully overwrite the global configuration.

![v4 1 8-failure](https://github.com/user-attachments/assets/9328c872-17dc-4800-a154-aae38f82be8b)

### Versions
- Plugin version: 4.1.8
- Craft version: 4.13.7

But I can successfully upload SEO Image on the old version of CMS.

![v3 4 37-success](https://github.com/user-attachments/assets/9efb0706-30ff-49f8-8340-fe151eb9e27f)

### Versions
- Plugin version: 3.4.37
- Craft version: 3.7.53.1

**Comment by khalwat** [comment]
Show me a screenshot of the SEO Settings field for the entry in question, after the import. Is the image assigned there?

**Comment by Owen-Guo** [comment]
These two screenshots are the global and content configurations on the failed cms.
![global-seo-config](https://github.com/user-attachments/assets/70092b18-cb01-4737-93e9-34794b8a4f59)
![entry-content-config](https://github.com/user-attachments/assets/8ec11c83-db5b-446c-97af-3174e5f62ed5)

**Comment by khalwat** [comment]
Yes, but you said you were importing the image into an SEO Settings field, yes? Can I get a screenshot of that?

**Comment by Owen-Guo** [comment]
Yes. Do you want a SEO screenshot of this Imported entry?
![17356142352717](https://github.com/user-attachments/assets/04f8e45d-49e5-4852-a112-e64d90ecc943)

**Comment by khalwat** [comment]
The **SEO Image Source** lightswitch is not set to override. Turn that on, and I think it'll work as you're expecting (assuming the asset was imported properly into that SEO Settings field).

**Comment by Owen-Guo** [comment]
Yes, you are right. But the problem is that I use the same json file, it can be uploaded successfully in the old version of the plugin, and lightswitch is overwritten. 
And I checked the feed me log and the server log and found no errors.

**Comment by khalwat** [comment]
That's a new feature in the latest version of SEOmatic, both for Craft 3 and for Craft 4. It requires that the the override lightswitch is activated for the override to happen.

If this was set before the import, it would override as expected.

**Comment by Owen-Guo** [comment]
OK, I will try to reconfigure the SEO fields and do a new round of testing.
